conf/cgroup.conf

```diff
@@ -0,0 +1,14 @@
+# use cgroup: IF_USE_CPU_CGROUP=0
+# not use cgroup: IF_USE_CPU_CGROUP=1 (default)
+IF_USE_CPU_CGROUP=1
```

Comment:
invert the configuration, or use `true`/`false` to explicitly tell whether use or not.